e2e/windows/nodebootstrapping_template.json

```diff
@@ -156,7 +156,6 @@
         "--cloud-provider": "azure",
         "--cluster-dns": "10.0.0.10",
         "--cluster-domain": "cluster.local",
-        "--dynamic-config-dir": "/var/lib/kubelet",
```

Comment:
Removed all things around dynamic kubelet config as it's deprecated. Logged onto v1.26.0 (or v1.26.3) failed deployment node and checked the `kubelet.err.log` saying `failed to set feature gates from initial flags-based : unrecognized feature gate: DynamicKubeletConfig`. Cse validation on these versions would fail if `DynamicKubeletConfig=false` is still there in the json file. 

Toyed around a bit with this and I decided to simply remove every occurence since it would still be compatible with earlier versions confirmed empirically.

FYI @junjiezhang1997